quickswitch.py

```diff
@@ -429,6 +448,11 @@ def main():
 
     args = parser.parse_args()
 
+    global follow
+    follow = args.follow
+    global followifempty
+    followifempty = args.followifempty
```

Comment:
I had the same misgivings. There's a lot of work to refactor to try to add those as arguments to `move_container_to_workspace` though.